index.js

```diff
@@ -48,6 +48,7 @@ function send(ctx, path, opts) {
     var maxage = opts.maxage || opts.maxAge || 0;
     var hidden = opts.hidden || false;
     var format = opts.format === false ? false : true;
+    var extensions = Array.isArray(opts.extensions) ? opts.extensions : false;
```

Comment:
Hmmm, I have seen this approach in express also and it would not be a bad idea to use over here but after a while I'm looping through extensions array so this would add unneeded additional O(n) complexity.